### PR TITLE
[conn] Add connectivity checks for resets, rst_en, and cg_en

### LINKED ISSUE #10039: [dv] Create clock and reset connectivity test for all IPs in top-level
Use FPV connectivity framework, so that we can eliminate this kind of SW tests.

@sriyerg @cindychip

### PR DESCRIPTION
**Pull request by matutem**
Specifically check connectivity of rstmgr resets_o and rst_en_o outputs, and clkmgr cg_en_o.
Also fix some connectivity failures and minor typos.

Fixes #10039

Signed-off-by: Guillermo Maturana <maturana@google.com>

### CODE REVIEW

File: hw/top_earlgrey/formal/conn_csvs/clkmgr_peri.csv [hunk 1]
```diff
@@ -10,6 +10,8 @@
 # clkmgr peri clock connectivity
 CONNECTION, CLKMGR_PERI_CLK_ADC_CTRL_CLK, top_earlgrey.u_clkmgr_aon, clocks_o.clk_io_div4_peri, top_earlgrey.u_adc_ctrl_aon, clk_i
 
+CONNECTION, CLKMGR_PERI_CLK_AST_USB_CLK,   top_earlgrey.u_clkmgr_aon, clocks_o.clk_usb_peri,     u_ast, clk_ast_usb_i
```
Comment (matutem) [inline hw/top_earlgrey/formal/conn_csvs/clkmgr_peri.csv:13]: This and clkmgr_secure.csv line 20 fix a pre-existing failure.

File: hw/top_earlgrey/formal/conn_csvs/clkmgr_secure.csv [hunk 2]
```diff
@@ -35,7 +34,7 @@ CONNECTION, CLKMGR_SECURE_KMAC_CLK_LC_CTRL, top_earlgrey.u_clkmgr_aon, clocks_o.
 CONNECTION, CLKMGR_SECURE_CLK_OTP_CTRL_CLK,     top_earlgrey.u_clkmgr_aon, clocks_o.clk_io_div4_secure, top_earlgrey.u_otp_ctrl, clk_i
 CONNECTION, CLKMGR_SECURE_CLK_OTP_CTRL_EDN_CLK, top_earlgrey.u_clkmgr_aon, clocks_o.clk_main_secure,    top_earlgrey.u_otp_ctrl, clk_edn_i
 
-CONNECTION, CLKMGR_SECURE_CLK_PWRMGR_CLK, top_earlgrey.u_clkmgr_aon, clocks_o.clk_io_div4_secure, top_earlgrey.u_pwrmgr_aon, clk_esc_i
+CONNECTION, CLKMGR_SECURE_CLK_PWRMGR_CLK, top_earlgrey.u_clkmgr_aon, clocks_o.clk_io_div4_secure, top_earlgrey.u_pwrmgr_aon, clk_lc_i
```
Comment (matutem) [inline hw/top_earlgrey/formal/conn_csvs/clkmgr_secure.csv:37]: This was another failure.

**Review by cindychip — APPROVED** [review]
LG! Could you also check if the conn_testplan is updated?

**Comment by matutem** [comment]
The testplan has been updated.

**Pull request merged**